File: templates/java-paper/src/main/java/com/example/plugin/MyPlugin.java
package __BASE_PACKAGE__;

import org.bukkit.plugin.java.JavaPlugin;

	public final class __PLUGIN_NAME__ extends JavaPlugin {
	@Override
	public void onEnable() {
			getLogger().info("__PLUGIN_NAME__ enabled!");
			if (getCommand("__PLUGIN_NAME__") != null) {
				getCommand("__PLUGIN_NAME__").setExecutor((sender, command, label, args) -> {
			sender.sendMessage("Hello from __PLUGIN_NAME__!");
			return true;
				});
			}
	}

	@Override
	public void onDisable() {
		getLogger().info("__PLUGIN_NAME__ disabled!");
	}
}

File: templates/kotlin-paper/src/main/kotlin/com/example/plugin/MyPlugin.kt
package __BASE_PACKAGE__

import org.bukkit.plugin.java.JavaPlugin

class __PLUGIN_NAME__ : JavaPlugin() {
	override fun onEnable() {
		logger.info("__PLUGIN_NAME__ enabled!")
		getCommand("__PLUGIN_NAME__")?.setExecutor { sender, _, _, _ ->
			sender.sendMessage("Hello from __PLUGIN_NAME__!")
			true
		}
	}

	override fun onDisable() {
		logger.info("__PLUGIN_NAME__ disabled!")
	}
}


